apps/core/components/ItineraryCard/ItineraryCardWrapper.web.js

```diff
@@ -36,16 +42,21 @@ const ItineraryCardWrapper = ({ localizedPrice, children, layout }: Props) => {
         )}
         <View style={styles.sectors}>{children}</View>
         {!mobileLayout && (
-          <Icon name="chevron-down" color={defaultTokens.paletteInkLighter} />
+          <Icon
+            name={detailOpen ? 'chevron-up' : 'chevron-down'}
+            color={defaultTokens.paletteInkLighter}
+          />
         )}
       </View>
       {mobileLayout && (
         <View style={styles.footer}>
           <LocalizedPrice localizedPrice={localizedPrice} />
           <Button
-            label="Show Detail"
+            label={detailOpen ? 'Hide' : 'Show Detail'}
             type="secondary"
-            rightIcon={<Icon name="chevron-down" />}
+            rightIcon={
+              <Icon name={detailOpen ? 'chevron-up' : 'chevron-down'} />
```

Comment:
`detailOpen ? 'chevron-up' : 'chevron-down'` is used twice in render, what about separate it to `const`?

Reply:
Good idea. Extracted to `const`.